src/type/types.cpp

```diff
@@ -33,10 +33,11 @@ int DEFAULT_TUPLES_PER_TILEGROUP = 1000;
 int TEST_TUPLES_PER_TILEGROUP = 5;
 
 // For threads
-size_t QUERY_THREAD_COUNT = 1;
+size_t CONNECTION_THREAD_COUNT = 1;
```

Comment:
for now, we don't have any file for keeping these contants. we should create one.